### PR TITLE
utilize global colors for vars

### LINKED ISSUE #181: Enhancement: Add action button with kebab icon to the kebab menu
The kebab menu can be seen here: https://pf-next.netlify.com/components/Dropdown/examples/ It still requires introducing the action button as a result of issue #274. Icon for kebab should use https://fontawesome.com/icons/ellipsis-v?style=solid

Component lifecycle: 
- [x] Interaction design requirements 
- [x] Visuals 
- [x] First checkpoint complete and approved 
- [ ] CSS 
- [ ] React
- [ ] Second checkpoint complete and approved 

Contacts: 
- IxD @mcarrano 
- Visual @kybaker 
- CSS

### LINKED ISSUE #288: Change harded colors on btn to be global variables
The button component has hardcoded color variables, they should be global variables.

### LINKED ISSUE #378: Bug: missing hover states on dropdowns
After merging #286 we lost hover states on the dropdowns

### PR DESCRIPTION
**Pull request by jlmitch5**
closes #288 

updates buttons usage of variables to all be globals (including stuff defined in placeholders)

Fixes #378 
Closes #379 
Closes #181

### CODE REVIEW

**Comment by andresgalante** [comment]
I'd like to merge this one after #341

It'll probably generate some conflicts, I'll rebase this PR when we do merge #341

**Comment by andresgalante** [comment]
Even though it's not done, I've started #364 on this PR

File: src/patternfly/components/Button/styles.scss [hunk 1]
```diff
@@ -69,29 +70,29 @@
   // Tertiary btn
   --pf-c-button--m-tertiary--BackgroundColor: transparent;
   --pf-c-button--m-tertiary--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-button--m-tertiary--Color: var(--pf-color-black-900);
+  --pf-c-button--m-tertiary--Color: var(--pf-global-Color-100);
```
Comment (dmiller9911) [inline src/patternfly/components/Button/styles.scss]: Missing some dashes I think: `--pf-global-Color-100` vs `--pf-global--Color--100`

File: src/patternfly/components/Button/styles.scss [hunk 2]
```diff
@@ -69,29 +70,29 @@
   // Tertiary btn
   --pf-c-button--m-tertiary--BackgroundColor: transparent;
   --pf-c-button--m-tertiary--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-button--m-tertiary--Color: var(--pf-color-black-900);
+  --pf-c-button--m-tertiary--Color: var(--pf-global-Color--100);
```
Comment (dmiller9911) [inline src/patternfly/components/Button/styles.scss]: whomp whomp still missing one between `global` and `Color` `--pf-global--Color--100` Wonder if there is a utility to verify these.  I might take a look to see if I can find one.

**Pull request merged**